### PR TITLE
Rename labelComponent to label for scatter & point

### PR DESCRIPTION
**Pull request by cpb**
See discussion here for why: FormidableLabs/victory-chart/pull/101

### CODE REVIEW

**Comment by cpb** [comment]
mm hang on i haven't actually checked this myself

**Comment by cpb** [comment]
:+1:

File: src/components/victory-scatter/point.jsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ export default class Point extends React.Component {
       x: React.PropTypes.any,
       y: React.PropTypes.any
     }),
-    labelComponent: React.PropTypes.element,
+    label: React.PropTypes.element,
```
Comment (boygirl) [inline src/components/victory-scatter/point.jsx]: Should this be `labels` everywhere. I guess it's possible that someone might want to pass in an array of labels for each scatter. Either way, I'd like to be consistent with the other chart types.

Comment (cpb) [inline src/components/victory-scatter/point.jsx]: Hey just saw this now. I'll fix it in the yet-to-be-merged https://github.com/FormidableLabs/victory-chart/pull/117

Comment (cpb) [inline src/components/victory-scatter/point.jsx]: How would you like it to behave with an array of labels?

**Comment by boygirl** [comment]
I could imagine someone passing in an array of strings to the `labels` prop rather than merging them into an existing data object. Passing in an array of label components is a little more contrived, but I guess someone could want that?

**Comment by boygirl** [comment]
What if `renderPoint` checks for `labels[index]` and passes that on if it exists?

**Comment by boygirl** [comment]
@cpb @divmain just added a `matchDataLength` prop validator, which would be useful here too.

**Comment by coopy** [comment]
See this example for `matchDataLength` PropType validation usage:
https://github.com/FormidableLabs/victory-chart/pull/138/files#diff-9f67a6d48df60bafddfc9e6c03245605R62

**Pull request closed without merging**